AirCasting/SessionViews/SessionCartViewModel/MeasurementsViewModel.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 11/09/2021.
+//
+
+import Foundation
+import CoreLocation
+import CoreData
+
+protocol SyncMeasurements {
```

Comment:
`SyncMeasurements` is not a great name. It's usually a convention that imperative verbs are used for function names (like `doThis()`, `doThat()`). For protocols we use nouns or adjectives (which IMO better reflect what protocol actually is, but sometimes there is no easy adjective). Let's rename it? Perhaps make it a protocol for whole VM instead of just this sync function? So we don't directly use `CardMeasurementsViewModel` in a view. Examples can be found in code on how to do that (very tiny generics syntax is involved) like `ForgotPasswordViewModel`